frontend/packages/knative-plugin/src/catalog/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { TFunction } from 'i18next';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+export const getEventSourceSupport = (item: K8sResourceKind, t: TFunction) => {
```

Comment:
We could avoid passing `t: TFunction` as a second argument and straight away import and use i18next.t
i.e i18next.t('knative-plugin~Tech Preview')